docs/migration.md

```diff
@@ -0,0 +1,121 @@
+# Lisk V4.x Migration guide
+
+This section explains how to migrate a Lisk Core v3.x node to Lisk Core v4.x using the Lisk migrator.
+
+The Lisk migrator CLI tool will generate a new genesis block for Lisk Core v4.
+The new genesis block is created based on a snapshot of the old blockchain (running on Lisk Core v3.x) at a given height.
+
+> Note: All actively forging validators on the Lisk Mainnet and the Lisk Testnet need to follow this guide to correctly migrate their nodes to the new network, in order to not miss any blocks after the network hard fork.
+
+Please ensure you are running version 3.x of Lisk Core to be able to seamlessly migrate to Lisk Core 4.x.
+
+## Setting up the Lisk migrator
+
+### Download checksum and verify
+
+Download the checksum and verify the successful download of lisk-migrator.
+
+```
+curl -o lisk-migrator-v2.0.0.SHA256 https://downloads.lisk.com/lisk-migrator/lisk-migrator-v2.0.0.SHA256
+```
+
+### Extract and add to PATH
+
+```
+tar -xf ./lisk-migrator-v2.0.0.SHA256
```

Comment:
Seems incorrect. There's no need to extract the checksum file. It must be used to validate the binary checksum.

On Linux:
```
sha256sum -c genesis_block.json.tar.gz.SHA256 genesis_block.json.tar.gz
genesis_block.json.tar.gz: OK
sha256sum: genesis_block.json.tar.gz: no properly formatted SHA256 checksum lines found
```

On Mac:
```
shasum -a 256 -c genesis_block.json.tar.gz.SHA256 genesis_block.json.tar.gz
genesis_block.json.tar.gz: OK
shasum: genesis_block.json.tar.gz: no properly formatted SHA checksum lines found
```